src/main/java/backend/Logic.java

```diff
@@ -231,11 +232,34 @@ public void removeUnusedModels(Set<String> reposInUse) {
                 .forEach(models::removeRepoModelById);
     }
 
-    public ImmutablePair<Integer, Long> updateRemainingRate(ImmutablePair<Integer, Long> rateLimits) {
+    /**
+     * Updates the UI components that are using GitHub rate limits information.
+     * @param rateLimits The GitHub API rate limits information.
+     *                   rateLimits.getLeft() contains the number of API requests remaining
+     *                   in the current rate limit window.
+     *                   rateLimits.getRight() contains the time at which the current
+     *                   API rate limit window resets in UTC epoch milliseconds.
```

Comment:
Let's abstract this parameter into a new class called `RefreshRateInfo`. That way, this description can be done in one place (i.e. header comment of that class).